applications/openshift/controller/controller_rotate_kubelet_server_certs/rule.yml

```diff
@@ -37,19 +37,21 @@ rationale: |-
     ensures that there are no downtimes due to expired certificates and thus
     addressing the availability in the C/I/A security triad.
 
+    In OpenShift 4, the kubelete certification rotation is enabled by default, hence this check is not-applicable.
```

Comment:
As far as I recall, this auto-rotation has been there from the inception of OpenShift 4, even this [document](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.4/html/security/certificate-types-and-descriptions#cert-types-node-certificates) says :`Once the cluster is installed, the node certificates are auto-rotated.`  `The kubelet-managed certificate is valid for one year and rotates automatically at around the 80 percent mark of that one year`. I don't see this section in <4.4 document, but I recall that we were never required to tweak it through feature-gate parameter.

I also agree that this rule can be removed since we already have some kubelet rules in place, which correctly checks for cert-rotation setting from kubelet config itself.

Reply:
Leave this note in.